testcafe/tests/profile-page-test.js

```diff
@@ -7,6 +7,13 @@ const adminUser = Role('http://127.0.0.1:8080/', async t => {
     .click("#login-loginbutton")
 });
 
+const adminUserAlternative = Role('http://127.0.0.1:8080/', async t => {
+    await t
+    .typeText('#input-18', 'Admin')
+    .typeText('#input-21', 'Admin123')
```

Comment:
I guess "#input-18" and "#input-21" are the name and password field?
If so it would make sense to set the input-fields id`s to a more meaningful name (like you did below)

Reply:
We have to fix this everywhere, Would suggest another pr.